### ISSUE TITLE
[Package Issue]: SweetScape.010Editor

### ISSUE THREAD

**Issue by assarbad** [microsoft/winget-pkgs#91645]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

I had version 13.0 (13.0.0) installed as 64-bit version. It showed up in `winget upgrade` like so:

```
>winget upgrade
Name                                             Id                                     Version       Available     Source
--------------------------------------------------------------------------------------------------------------------------
010 Editor 13.0 (64-bit)                         SweetScape.010Editor                   13.0          13.0.1        winget
```

Due to the fact that it's not possible to pin packages, I tried to upgrade it by ID, i.e. with:

```
>winget upgrade --id SweetScape.010Editor
Found 010 Editor [SweetScape.010Editor] Version 13.0.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.sweetscape.com/010EditorWin32Installer13.0.1.exe
  ██████████████████████████████  17.7 MB / 17.7 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

I'm sure you can spot the issue. As I wrote I had the old version as 64-bit and the new one "upgrades" to 32-bit, i.e. now I have two versions installed in parallel.

### Steps to reproduce

* Have the 64-bit 13.0.0 version installed
* Instruct winget to upgrade this particular package
* Get the 32-bit version installed in parallel to the existing 64-bit version

### Actual behavior

Installs 32-bit version

### Expected behavior

Should upgrade the existing 64-bit version

### Environment

```shell
>winget --info
Windows Package Manager v1.3.2691
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.2364
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2691.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

### Screenshots and Logs

The issue has been demonstrated above.

**Comment by assarbad** [comment]
It's clear why this happens, the `Architecture:` key is wrong. The following patch would fix it:

```diff
diff --git a/manifests/s/SweetScape/010Editor/13.0.1/SweetScape.010Editor.installer.yaml b/manifests/s/SweetScape/010Editor/13.0.1/SweetScape.010Editor.installer.yaml
index 89255a879f..0505a22515 100644
--- a/manifests/s/SweetScape/010Editor/13.0.1/SweetScape.010Editor.installer.yaml
+++ b/manifests/s/SweetScape/010Editor/13.0.1/SweetScape.010Editor.installer.yaml
@@ -19,11 +19,11 @@ FileExtensions:
 - s37
 ReleaseDate: 2022-09-07
 Installers:
-- Architecture: x86
+- Architecture: x64
   InstallerUrl: https://download.sweetscape.com/010EditorWin64Installer13.0.1.exe
   InstallerSha256: 4C55D604397DB9789B0EB39042707CBCF0BD36D5B43910F47462955B2B53332F
   ProductCode: 010 Editor_is1
-- Architecture: x64
+- Architecture: x86
   InstallerUrl: https://download.sweetscape.com/010EditorWin32Installer13.0.1.exe
   InstallerSha256: 832F548A0A8850D353E3FC67D7C65301883F07F035131AA77D3144653A4C40E5
   ProductCode: 010 Editor_is1
```

Going to submit a pull request.

@vedantmgoyal2009 I think something may be wrong in the bot. Just FYI!

**Comment by assarbad** [comment]
Fixed by #91647
